Update community-grants.md

giacastello opened the pull request:
Adding the Decentraland DAO Grants Budgeting System defined in governance proposal to the Community Grants page.

menduz commented:
What is that parcel ID? `1.15792089237316195423570985008687907837276713420356456256678977458620023701475e+77`

nachomazzara commented:
> What is that parcel ID? `1.15792089237316195423570985008687907837276713420356456256678977458620023701475e+77`
> 
> If the server can resolve that number it is far from correct.

Is the way we store the parcel `token_id` in our DB. I removed it from the documentation in order to avoid error-prone.

menduz commented:
are we storing `token_id` as a varchar or as a number? if it is a number we should change it, because DB numbers goes to 64bit and that number is a 256bit number

nachomazzara commented:
> are we storing `token_id` as a varchar or as a number? if it is a number we should change it, because DB numbers goes to 64bit and that number is a 256bit number

As a `text` (varchar).

menduz commented:
So we have a .toNumber convertion somewhere. look at the end of the number `1.15...75e+77` and back to text again @nicosantangelo FYI

nachomazzara commented:
> So we have a .toNumber convertion somewhere. look at the end of the number `1.15...75e+77` and back to text again @nicosantangelo FYI

Right, we are aware of that. We use it just for internal reference. We don't expose that to the users.

menduz commented:
sabe

nachomazzara commented:
@menduz @nearnshaw can we merge it?

nearnshaw commented:
👍  fine by me
it looks like @nicosantangelo is set as a reviewer, so we need him to sign before we can merge

nachomazzara commented:
Oh, yes I set him. Ping @nicosantangelo

nicosantangelo commented:
LGTM !

nearnshaw commented:
@nachomazzara looks like one of the commits wasn't signed with a youbikey :/
That means I can't merge this myself, we need to override it manually

@eordano or @adrabenche could you please merge this branch with an override of the security checks?

nicosantangelo commented:
I can do it

giacastello committed:
Update community-grants.md

Adding the Decentraland DAO Grants Budgeting System defined in governance proposal to the Community Grants page.

Signed-off-by: Gia ░░░░░░░░░░░░▒▒▒▒▒▒▒▒▒▒▒▒ Castello <giacastello@gmail.com>

yemel committed:
Merge branch 'main' into patch-7

giacastello committed:
Merge branch 'main' into patch-7

giacastello committed:
Merge branch 'main' into patch-7

giacastello committed:
Update community-grants.md

Adding official mediums and English required for Awesome Repository and Decentraland Documentation contributions.

giacastello committed:
Merge branch 'main' into patch-7
